sass/index.scss

```diff
@@ -24,3 +24,4 @@
 @import "pages/common/MultiSelectRenderer";
 @import "pages/common/PageTreeSelector";
 @import "pages/common/PageForm";
+@import "pages/settings/Page-settings.scss";
```

Comment:
Please rename this file as PageSettings.scss, using PascalCase as all the other files